Lib/distutils/command/install.py

```diff
@@ -3,6 +3,7 @@
 Implements the Distutils 'install' command."""
 
 import sys
+import sysconfig
```

Comment:
The `sysconfig` saga is a long one.  Unfortunately, there are still two versions of `sysconfig`, the original one in `Lib/distutils` and the newer, stand-alone one in `Lib`.   There is a clean separation between the two in that stuff within `Lib/distutils` only uses its version and outside of `Lib/distutils` only the `Lib/sysconfig.py` version is used.  It's also made more complicated because some third-party projects modify monkeypatch or otherwise have their own modified versions of `distutils`.  So please don't add a dependency like this.  This is one case where unfortunately DRY does not apply.